Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -392,6 +393,9 @@ bool VideoBackendBase::InitializeShared(std::unique_ptr<AbstractGfx> gfx,
   g_Config.VerifyValidity();
   UpdateActiveConfig();
 
+  if (g_Config.bDumpTextures)
+    OSD::AddMessage(fmt::format("Texture Dumping: Enabled"), OSD::Duration::NORMAL);
```

Comment:
Maybe to make the problem clearer, we could have this message say something like this?

```
Texture dumping is enabled. This will reduce performance.
```